honeybadgermpc/passive.py

```diff
@@ -168,6 +203,36 @@ def __mul__(self, other): raise NotImplemented
 
         def __str__(self): return '{%d}' % (self.v)
 
+    def _binopShare(fut, other, op):
```

Comment:
I see that the `_binopShare` above has an `isinstance` check for `ShareFuture`. Do we still need this method?